src/pages/cluster/ClusterGroupForm.tsx

```diff
@@ -85,7 +86,17 @@ const ClusterGroupForm: FC<Props> = ({ group }) => {
         .then(() => {
           const verb = group ? "saved" : "created";
           navigate(`/ui/cluster/group/${values.name}`);
-          toastNotify.success(`Cluster group ${values.name} ${verb}.`);
+          toastNotify.success(
+            <>
+              Cluster group{" "}
+              <ResourceLink
+                type="cluster-member"
+                value={values.name}
+                to={`/ui/cluster/group/${values.name}`}
+              />{" "}
```

Comment:
A cluster group is a defined set of cluster members, not a cluster member itself. I think we do need a different resource type for it. Maybe also a different icon?

Reply:
I think in this case we can just use the cluster icon instead? Will confirm with @piperdeck

Reply:
Mapped cluster groups to the `cluster-host` icon for now as discussed.